src/types/image.d.ts

```diff
@@ -27,12 +27,14 @@ export interface ImportImage {
 export interface RemoteImage {
   aliases: string;
   arch: string;
+  created_at: number;
   lxd_requirements?: {
     secureboot: boolean;
   };
   os: string;
+  pool?: string;
   release: string;
-  release_title?: string;
+  release_title: string;
```

Comment:
It's not always there, see the other comment. This needs to be treated as optional.

Reply:
Huh, this shouldn't have ended up in this branch, I'll reset the head to be without the commit about release_title.

Reply:
Might still be worth noting it down and fix it in the other branch.

Reply:
Yes, I changed it there to have it fixed.